### PR TITLE
Proof read STABLE_ORBIT

### LINKED ISSUE #258: Proof STABLE_ORBIT
Proof read transcribed code of [STABLE_ORBIT](//github.com/chrislgarry/Apollo-11/blob/master/Comanche055/STABLE_ORBIT.agc) against scans

Lines: 429
Page: 0525—0532

Reduced quality scans can be found [here][1]

[1]://ibiblio.org/apollo/ScansForConversion/Comanche055

### PR DESCRIPTION
**Pull request by Feliix42**
This should close #258.

Some content fixes here and there (words missing/added), the rest were mostly indentation/spacing and period/comma fixes. 

The question I raised in #258 remains, still. ^^

### CODE REVIEW

File: Comanche055/STABLE_ORBIT.agc [hunk 1]
```diff
@@ -31,7 +31,7 @@
 # STABLE ORBIT RENDEZVOUS PROGRAMS (P38 AND P78)
 #
 # MOD NO -1		LOG SECTION - STABLE ORBIT - P38-P39
-# MOD BY RUDNICKI.S	DATE 25JAN68
+# MOD BY RUDNICKI, S	DATE 25JAN68
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ❌ This is a dot without a space

File: Comanche055/STABLE_ORBIT.agc [hunk 2]
```diff
@@ -56,7 +56,7 @@
 #
 #		(A)  TIG	TIME OF SOI MANEUVER
 #		(B)  CENTANG	ORBITAL CENTRAL ANGLE OF THE PASSIVE VEHICLE
-#				DURING THE TRANSFER FROM TIG TO TIME OF INTERCEPT
+#				DURING TRANSFER FROM TIG TO TIME OF INTERCEPT
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 3]
```diff
@@ -85,7 +85,7 @@
 #			THE SOI (SOR) MANEUVER
 #	(6)  DELVTPI	MAGNITUDE OF DELTA V AT SOI (SOR) TIME
 #	(7)  DELVTPF	MAGNITUDE OF DELTA V AT INTERCEPT TIME
-#	(8)  DELTA 	VELOCITY AT SOI (AND SOR) - LOCAL VERTICAL
+#	(8)  DELVLVC	DELTA VELOCITY AT SOI (AND SOR) - LOCAL VERTICAL
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 4]
```diff
@@ -122,21 +122,21 @@ P78		TC	AVFLAGP         # OTHER VEHICLE ACTIVE
 		TCF	GOTOP00H	# TERMINATE
 		TCF	+5		# PROCEED
 		TCF	-5		# RECYCLE
-		CAF	THREE		# IMMEDIATE RETURN -- BLANK R1, R2
+		CAF	THREE		# IMMEDIATE RETURN - BLANK R1, R2
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 5]
```diff
@@ -122,21 +122,21 @@ P78		TC	AVFLAGP         # OTHER VEHICLE ACTIVE
...
 		TCR	BLANKET
 		TCF	ENDOFJOB
 		CAF	FIVE
 		TS	OPTION1
 		CAF	ONE
 		TS	OPTION2		# OPTION CODE IS SET TO 1
 # Page 527
-		CAF	V04N06SR	# DISPLAY OPTION CODE -1 = SOI, 2 = SOR
+		CAF	V04N06SR	# DISPLAY OPTION CODE - 1 = SOI, 2 = SOR
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 6]
```diff
@@ -122,21 +122,21 @@ P78		TC	AVFLAGP         # OTHER VEHICLE ACTIVE
...
 		TCR	BANKCALL
 		CADR	GOFLASHR
 		TCF	GOTOP00H	# TERMINATE
 		TCF	+5		# PROCEED
 		TCF	-5		# RECYCLE
-		CAF	BIT3		# IMMEDIATE RETURN -- BLANK R3
+		CAF	BIT3		# IMMEDIATE RETURN - BLANK R3
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 7]
```diff
@@ -150,7 +150,7 @@ P78		TC	AVFLAGP         # OTHER VEHICLE ACTIVE
 			TINT
 		STORE	TINTSOI		# STORE FOR SOR PHASE
 		CLRGO
-			OPTNSW		# OPTNSW: ON = SOI. OFF = SOR
+			OPTNSW		# OPTNSW: ON = SOI, OFF = SOR
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️ `.` -> `,`

✨ `:` should be a `;` here

File: Comanche055/STABLE_ORBIT.agc [hunk 8]
```diff
@@ -186,7 +186,7 @@ RECYCLE		CALL
 			OPTN2
 			TINT
 		STCALL	TDEC1		# PRECISION UPDATE PASSIVE VEHICLE TO
-			INTRPVP		# INTERCEPT TIME
+			INTRPVP		#	INTERCEPT TIME
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 9]
```diff
@@ -209,7 +209,6 @@ JUNCTN2		DLOAD	DSU
 		STORE	TARGTIME	# TT = TI - DELTA T
 
 # .... MAINRTNE ....
-#
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 10]
```diff
@@ -248,7 +247,7 @@ MAINRTN1	VLOAD	ABVAL
 			VPASS4
 		VSU	ABVAL
 			VTPRIME
-		STOVL	DELVTPF		# DELTA V (FINAL) = V-T - VT
+		STOVL	DELVTPF		# DELTA V (FINAL) = V'T - VT
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 11]
```diff
@@ -287,7 +286,7 @@ DSPLY81		CAF	V06N81SR	# DISPLAY DELTA V (LV)
 #	CONDITIONS REQUIRED BY THE AGC TO MAKE A MIDCOURSE CORRECTION
 # Page 530
 #	MANEUVER AFTER COMPLETING THE SOI MANEUVER BUT BEFORE MAKING
-#	THE SOR MANEUVER.
+#	THE SOR MANEUVER
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 12]
```diff
@@ -298,16 +297,16 @@ DSPLY81		CAF	V06N81SR	# DISPLAY DELTA V (LV)
 #
 # INPUT
 #
-#	(1)  TPASS4	TIME OF INTERCEPT -- SAVED FROM P38/P78
-#	(2)  TARGTIME	TIME THAT PASSIVE VEHICLE IS AT INTERCEPT POINT --
+#	(1)  TPASS4	TIME OF INTERCEPT - SAVED FROM P38/P78
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 13]
```diff
@@ -298,16 +297,16 @@ DSPLY81		CAF	V06N81SR	# DISPLAY DELTA V (LV)
...
+#	(2)  TARGTIME	TIME THAT PASSIVE VEHICLE IS AT INTERCEPT POINT -
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 14]
```diff
@@ -298,16 +297,16 @@ DSPLY81		CAF	V06N81SR	# DISPLAY DELTA V (LV)
...
 #			SAVED FROM P38/P78
 #
 # OUTPUT
 #
 #	(1)  TRKMKCNT	NUMBER OF MARKS
 #	(2)  TTOGO	TIME TO GO
 #	(3)  +MGA	MIDDLE GIMBAL ANGLE
-#	(4)  DELVLVC	DELTA VELOCITY AT MID -- LOCAL VERTICAL COORDINATES
+#	(4)  DELVLVC	DELTA VELOCITY AT MID - LOCAL VERTICAL COORDINATES
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 15]
```diff
@@ -338,15 +337,14 @@ P39/P79B	RTB	DAD
 		STORE	TIG		# TIG = T (PRESENT) + PREPARATION TIME
 # Page531
 		STCALL	TDEC1		# PRECISION UPDATE ACTIVE AND PASSIVE
-			PRECSET		# 	VEHICLES TO TIG
+			PRECSET		#	VEHICLES TO TIG
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 16]
```diff
@@ -338,15 +337,14 @@ P39/P79B	RTB	DAD
...
 		CALL
 			S34/35.1	# GET UNIT NORMAL
 		DLOAD	GOTO
 			TARGTIME
 			MAINRTNE	# CALCULATE DELTA V AND DELTA V (LV)
 
 # .... PREC/TT ....
-#
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️

File: Comanche055/STABLE_ORBIT.agc [hunk 17]
```diff
@@ -380,7 +378,6 @@ PREC/TT		STQ	DLOAD
 			RTRN
 
 # .... INTRPVP ....
-#
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️ Removing empty line

✨ Remove empty line above `INTRPVP` ☝️

File: Comanche055/STABLE_ORBIT.agc [hunk 18]
```diff
@@ -401,7 +398,6 @@ OTHERV		CALL
 			RTRN
 
 # .... VNDSPLY ....
-#
```
Comment (wopian) [inline Comanche055/STABLE_ORBIT.agc]: ✔️ Removing empty line

✨ Remove empty line above `VNDSPLY` ☝️

**Comment by Feliix42** [comment]
Done ✔️ :)

**Pull request merged**